src/utils/common.ts

```diff
@@ -58,7 +58,7 @@ export const IS_DEMO_MODE = import.meta.env.MODE === 'demo';
 export const IS_TEST_MODE = import.meta.env.MODE === 'test';
 
 export function logDev(message: unknown, ...optionalParams: unknown[]) {
-  if (IS_DEVELOPMENT_BUILD && !IS_TEST_MODE) {
+  if (IS_DEVELOPMENT_BUILD) {
```

Comment:
It can help to debug when running test mode.  The tests on github run with production build so logs will get excluded. (Remember development / production builds, dev / prod / test / demo / jwdev modes)